scenarios/04_transitive_via_export/child/src/test/java/transitive_via_export/parent/ChildTest.java

```diff
@@ -0,0 +1,13 @@
+package transitive_via_export.parent;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class ChildTest {
+  @Test
+  public void print_hello_three_times() {
+    assertEquals("hellohello", new Child().hello2times());
```

Comment:
hello2times returns a boolean, not a string.

Reply:
nice catch. we don't have a build that fails yet, unfortunately :)

addressed.